samples/bluetooth/iso_combined_bis_and_cis/README.rst

```diff
@@ -0,0 +1,120 @@
+
+After programming the sample to the development kit and programming other samples to CIS Peripheral device and BIS sink device, perform the following steps to test it:
+
+1. |connect_terminal_specific|
+#. Reset the kits.
+#. Wait until the sample connected to CIS Peripheral device and established CIS connection.
+#. Observe packets transmitted by CIS Peripheral to be retransmitted over BIS and received by BIS Sink.
+
+
+Sample output
+==============
+
+[00:00:00.454,589] <inf> bt_hci_core: HW Platform: Nordic Semiconductor (0x0002)
+[00:00:00.454,620] <inf> bt_hci_core: HW Variant: nRF53x (0x0003)
+[00:00:00.454,650] <inf> bt_hci_core: Firmware: Standard Bluetooth controller (0x00) Version ... Build ...
+[00:00:00.456,329] <inf> bt_hci_core: Identity: FD:DD:D6:88:AF:D4 (random)
+[00:00:00.456,359] <inf> bt_hci_core: HCI: version 5.4 (0x0d) revision ..., manufacturer 0x0059
+[00:00:00.456,390] <inf> bt_hci_core: LMP: version 5.4 (0x0d) subver ...
+[00:00:00.456,390] <inf> app_main:
+Bluetooth ISO Combined BIS and CIS sample
+
+The sample demonstrates data transfer over Bluetooth ISO
+CIS and BIS using the following topology:
+
+┌------┐     ┌-------┐      ┌--------┐
+|      |ISO  |  CIS  | ISO  |        |
+| CIS  ├----►|Central├-----►|BIS Sink|
+|Periph|CIS  |+ BIS  | BIS  |        |
+|      |     |Source |      |        |
+└------┘     └-------┘      └--------┘
+  (1)           (2)            (3)
+The sample only operates as a device 2: Combined CIS Central + BIS Source
+Please, use other samples as a CIS Peripheral (TX) and BIS Sink (RX) devices.
+
+
+[00:00:00.456,420] <inf> app_main: Starting combined CIS Central + BIS Source
+[00:00:00.459,777] <inf> app_bis_cis: CIS central started scanning for peripheral(s)
+[00:00:00.593,078] <inf> app_bis_cis: Connected: FB:DC:75:C0:18:0A (random)
+[00:00:00.593,597] <inf> app_bis_cis: Connecting ISO channel
+[00:00:00.600,189] <inf> app_bis_cis: BIS transmitter started
+[00:00:00.607,879] <inf> app_iso_tx: ISO TX Channel connected
+[00:00:00.608,459] <inf> app_iso_tx: Sent SDU, counter: 0
+[00:00:00.741,577] <inf> app_iso_rx: ISO RX Channel connected
+[00:00:01.449,615] <inf> app_iso_tx: Sent SDU, counter: 100
+[00:00:01.731,140] <inf> app_iso_rx: Received SDU: 100, empty SDU: 0, missed SDU: 0
+[00:00:02.449,645] <inf> app_iso_tx: Sent SDU, counter: 200
+[00:00:02.731,140] <inf> app_iso_rx: Received SDU: 200, empty SDU: 0, missed SDU: 0
+[00:00:03.449,615] <inf> app_iso_tx: Sent SDU, counter: 300
+[00:00:03.731,140] <inf> app_iso_rx: Received SDU: 300, empty SDU: 0, missed SDU: 0
+[00:00:04.449,615] <inf> app_iso_tx: Sent SDU, counter: 400
+[00:00:04.731,140] <inf> app_iso_rx: Received SDU: 400, empty SDU: 0, missed SDU: 0
+[00:00:05.449,645] <inf> app_iso_tx: Sent SDU, counter: 500
+
+
+References
+***********
+
+For more information about the scheduling of different types of packets refer to the `SDC Scheduling Documentation`_.
```

Comment:
yes. What I wanted to say is:
Is it possible to solve this with a reference like :ref:`softdevice_controller_iso` instead of adding it to links.txt. I think that might be a more stable solution to the problem because:
1. The link would still work even if the URL changes
2. The link might be version specific and not always point to the "latest" version of the doc.

Reply:
Thank you, I misunderstood your initial comment. I replaced the link with `:ref:`softdevice_controller_scheduling``